native-messaging/check_config_win.py

```diff
@@ -0,0 +1,64 @@
+'''
+A Python script that attempts to check that the configuration of your
+nativeMessaging app is set up correctly
+
+Currently requires Python 3.
+
+If you find more issues with setting this up, let's see if we can add to this
+script.
+'''
+import json
+import os
+import re
+import winreg

```

Comment:
hi, I do get: 

```
PS D:\git\webextensions-examples\native-messaging> python.exe .\check_config_win.py
Traceback (most recent call last):
  File ".\check_config_win.py", line 13, in <module>
    import winreg
ImportError: No module named winreg
```

IMO some more info is needed about dependnecies.

Reply:
winreg is a standard Python 3 module, are you using Python 2? If so making this script work in Python 2 could be a seperate bug: https://docs.python.org/3/library/winreg.html

Reply:
did install python3, but still an error. ... I don't have access to win machine atm. so I'll post again tomorrow.